src/pages/settings/PasswordPage.js

```diff
@@ -143,15 +202,19 @@ class PasswordPage extends Component {
                                 autoCompleteType="password"
                                 textContentType="password"
                                 value={this.state.newPassword}
-                                onChangeText={newPassword => this.setState({newPassword})}
-                                onFocus={() => this.setState({isPasswordRequirementsVisible: true})}
-                                onBlur={() => this.onBlurNewPassword()}
+                                hasError={this.state.errors.newPassword}
+                                errorText=""
```

Comment:
This is not needed. We should default to an empty string if not passing this value.